tests/pthread_tls/tests/01-run.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+
+def _check_test_output(child):
+    child.expect('show tls values:')
+    for i in range(20):
+        if i != 5:
+            child.expect('key\[%d\]: [0-9]*, val: %d'
+                         % (i, i + 1 if i != 3 else 42))
+
+
+def testfunc(child):
+    child.expect('START')
+    child.expect('-= TEST 1 - create 20 tls with sequencial values 0...19 =-')
+    _check_test_output(child)
+    child.expect('-= TEST 2 - '
+                 'delete deliberate key \(key\[5\]:[0-9]{10}\) =-')
```

Comment:
can you replace the `{10}` with a `+` here please? No need to be so specific about the numbers here.